test/e2e/main_test.go

```diff
@@ -221,7 +169,7 @@ func testManualOLMUpgradeInstall(
 	purgeReports,
 	purgeReportDataSources bool,
 	testInstallFunction InstallTestCase,
-	testMeteringConfigSpec metering.MeteringConfigSpec,
+	testMeteringConfigManifestPath string,
```

Comment:
Does this need the `test` prefix? 

Also, this func name should be capitalized (not changing in this PR, but might as well fix)

Reply:
This is technically a test helper function to the top-level `TestManualMeteringInstall` function, so we don't want the function to be exported here. Something I can do in another commit or a follow-up is organized these functions better so we have a better idea of what's being run and why.

Reply:
👍